h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -45,7 +49,19 @@ describe('pdf-metadata', function () {
     });
 
     describe('#getUri', function () {
-      it('returns the URN-ified document fingerprint as its URI', function () {
+      it('returns the non-file URI', function() {
+        return pdfMetadata.getUri().then(function (uri) {
+          assert.equal(uri, 'fakeUrl');
+        });
+      });
+
+      it('returns the URN-ified document fingerprint as its URI when url is a file url', function () {
```

Comment:
nit. Should be consistent about `URL`, `URI`, `URN` etc. being in caps in test names